File: frontend/app/lib/core/domain/day_of_week.dart
enum DayOfWeek {
  mon,
  tue,
  wed,
  thu,
  fri;

  String get apiValue => name.toUpperCase();

  static DayOfWeek fromApi(String value) => DayOfWeek.values.firstWhere((d) => d.apiValue == value);

  String get koreanLabel {
    switch (this) {
      case DayOfWeek.mon:
        return '월';
      case DayOfWeek.tue:
        return '화';
      case DayOfWeek.wed:
        return '수';
      case DayOfWeek.thu:
        return '목';
      case DayOfWeek.fri:
        return '금';
    }
  }
}

File: backend/app/src/main/java/com/smash/domain/group/DayOfWeek.java
package com.smash.domain.group;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum DayOfWeek {
    MON("월"), TUE("화"), WED("수"), THU("목"), FRI("금");

    private final String label;
}
